src/textual/widgets/_markdown.py

```diff
@@ -316,17 +318,59 @@ def compose(self) -> ComposeResult:
         self._blocks.clear()
 
 
+class MarkdownTableContent(Widget):
+    DEFAULT_CSS = """
+    MarkdownTableContent {
+        width: 100%;
+        height: auto;
+
+    }
+    MarkdownTableContent > .markdown-table--header {
+        text-style: bold;
+
+    }
+    MarkdownTableContent > .markdown-table--lines {
+
+    }
```

Comment:
Are empty `DEFAULT_CSS` rules needed to make component styles work?

Reply:
No, just a side-effect of editing. Will remove.